### PR TITLE
Portworx documentation and setup, modified kafka-perf for test consis…

### PR DESCRIPTION
**Pull request by NathanielRose**
…tency

### CODE REVIEW

**Comment by NathanielRose** [comment]
Closing

File: kafka-perf/commands.sh [hunk 1]
```diff
@@ -25,10 +25,19 @@ fi
 
 
 # Create topics
-kubectl exec -it kafkaclient-0 -- bin/kafka-topics.sh --zookeeper $ZOOKEEPER_NAME.$NAMESPACE:2181 --create --topic test-rep-one --partitions 6 --replication-factor 1
-kubectl exec -it kafkaclient-0 -- bin/kafka-topics.sh --zookeeper $ZOOKEEPER_NAME.$NAMESPACE:2181 --create --topic test --partitions 6 --replication-factor 3 
+# kubectl exec -it kafkaclient-0 -- bin/kafka-topics.sh --zookeeper $ZOOKEEPER_NAME.$NAMESPACE:2181 --create --topic test-rep-one --partitions 6 --replication-factor 1
+# kubectl exec -it kafkaclient-0 -- bin/kafka-topics.sh --zookeeper $ZOOKEEPER_NAME.$NAMESPACE:2181 --create --topic test --partitions 6 --replication-factor 3
```
Comment (cnadolny) [inline kafka-perf/commands.sh:29]: Why are these commented out? Can we remove or move them to a different file?

Comment (NathanielRose) [reply kafka-perf/commands.sh:29]: Removed

**Comment by cnadolny** [comment]
LGTM!

**Pull request merged**